edit/bzlmod/bzlmod_test.go

```diff
@@ -66,6 +66,9 @@ prox9 = use_extension(
 prox9.use(label = "@name//:bar")
 prox10 = use_extension("@dep//:extensions.bzl", "other_ext", dev_dependency = bool(1))
 prox10.use(dict = {"foo": "bar"})
+prox11 = use_extension("extension.bzl", "ext")
+prox12 = use_extension(":extension.bzl", "ext")
+prox13 = use_extension("//:extension.bzl", "ext")
```

Comment:
what about `use_extension("@repo_name//:extension.bzl", "ext")`?

Reply:
I added more tests covering this case, is that what you had in mind?